README.md

```diff
@@ -1,11 +1,15 @@
-
-## Página para gestionar el alquiler de pisos de estudiantes.
+# App para gestionar las peñas deportivas
```

Comment:
No se os pide que describáis un app, sino una idea.
Aparte, no has usado nunca la clave pública, como pone en la captura. Aunque es aceptable que uséis la web, conviene que empecéis a usar git desde la línea de órdenes, no podéis estar siempre así.